### DISCUSSION TITLE
Trying to migrate from Verdaccio 3.x to New 5.x Instance - Packages Don't Show on Web UI

### DISCUSSION THREAD

**Discussion by SSIJay** [verdaccio/verdaccio#3231] [Q&A]
Hello,

We decided internally we wanted to move to Verdaccio 5 after we had a back end hardware issue with version 3. We have a linux server running a docker container in AWS that appears to be working correctly as we can access it via the web and log into it using SSO.

However, after logging in, we do not see any packages even though we have copied them over from the existing version 3 server. I'm guessing we missed something in the either config or the Dockerfile creation that differs from version 3 to version 5, but we are struggling to find what that problem is.

Any help and suggestions would be greatly appreciated. I can provide more details on our new server setup if they are needed.

Thanks!

**Comment by juanpicado** [comment]
Have you seen?

5.x https://verdaccio.org/blog/2021/04/14/verdaccio-5-migration-guide#after-upgrade-i-dont-see-packages-on-the-ui
4.x https://verdaccio.org/blog/2019/05/13/the-new-docker-image-verdaccio-4#whats-new

**Reply by SSIJay** [reply]
Yes, we had seen the above. The format is confirmed to be what 5.x recommends. We are able to push packages to this new Verdaccio instance. We are able to curl to see the packages. We are able to use npm commands to pull. We copied the list from the .sinopia-db.json in version 3 to the .verdaccio-db.json in version 5. It all seems to be operational via cli. We just are seemingly having issue with the web UI actually listing and finding the packages when it is accessed.

**Reply by juanpicado** [reply]
What does return the browser? Any error on chrome console ?

**Reply by SSIJay** [reply]
We just get the No Package Published Yet. splash screen. No errors in the console.

**Reply by juanpicado** [reply]
I'd recommend add `DEBUG=verdaccio*` as env variable and see if you notice something weird, also increase logging to `debug` in the logger.

Maybe give you some hint, if there is nothing being returned by the web packages endpoint is because probably is not able to find the storage and has created another one. 

https://github.com/verdaccio/verdaccio/wiki/Debugging-Verdaccio#debugging-with-debug

**Reply by SSIJay** [reply]
Getting back to this, we are running Verdaccio in a Docker container. Am I reading the debug documentation correctly as it says that there isn't a known way to debug in a Docker container?
